index.js

```diff
@@ -18,7 +18,9 @@ var DEFAULTS = {
         addNotifyTrigger: 'add_notify_trigger_to_table',
         removeNotifyTrigger: 'remove_notify_trigger_to_table'
     },
-    operationEvents: true
+    operationEvents: true,
+    checkUpdates: true, // only notify if update changes record, can be set to false if record comparison is expensive
+    sendRecordId: false // send only record IDs, true - to send column 'id', string to send column with a given name
```

Comment:
I like this. Great idea! I think in the future we should abstract this even more, maybe have an `columns` options where only specified columns are sent. For example: `columns: ['id', 'name', email']`